TreasureData/TreasureData.m

```diff
@@ -394,6 +413,18 @@ - (void)disableServerSideUploadTimestamp {
     self.serverSideUploadTimestampColumn = nil;
 }
 
+- (void)enableAutoAppendRecordUUID {
+    self.autoAppendRecordUUIDColumn = @"record_uuid";
+}
+
+- (void)enableAutoAppendRecordUUID: (NSString*)columnName {
+    self.autoAppendRecordUUIDColumn = columnName;
```

Comment:
I am not sure how possible it is in Obj-c to set a `NSString` value with a `null` value but we could protect/validate from that.
Same for 'empty string' and/or strings that don't follow the naming convention of TD.